src/Microsoft.Framework.PackageManager/Reports.cs

```diff
@@ -10,6 +10,26 @@ public class Reports
         public IReport Quiet { get; set; }
         public IReport Error { get; set; }
 
+        public void WriteInformation(string message)
+        {
+            Information.WriteLine(message);
+        }
+
+        public void WriteVerbose(string message)
+        {
+            Verbose.WriteLine(message);
+        }
+
+        public void WriteError(string message)
+        {
+            Error.WriteLine(message.Red());
+        }
+
+        public void WriteWarning(string message)
+        {
+            Information.WriteLine(message.Yellow());
+        }
+
```

Comment:
We want to use these everywhere, not just here. Today everyone does it in their own way